content/docs/contributing/docs.md

```diff
@@ -0,0 +1,14 @@
+# Contributing to the Documentation
+
+We welcome any contributions to our documentation repository,
+[cml-website](https://github.com/iterative/cml-website). Contributions can be
+updates to the documentation content, or (rare) changes to the JS engine we use
+to run the website.
+
+In case of a minor change, you can use the **Edit on GitHub** button to open the
+source code page. Use thethe Edit button (pencil icon) to edit the file
+in-place, and then **Commit changes** from the bottom of the page.
+
+> Please see our
```

Comment:
Make it a regular paragraph? Start with "For full details .." or "Otherwise ..."

Reply:
yes there's a bit of odd phrasing as well copied over from dvc.org. Will fix in both places.